Header header

float32 distance_m
float32 velocity_ms
float32 acceleration_mss
float32 raw_data_word
